src/components/App.css

```diff
@@ -0,0 +1,6 @@
+.app {
+  display: flex;
+  height: 700px;
+  width: 400px;
+  flex-direction: column;
+}
```

Comment:
### [required change]
- Width should be `700px`, not `400px` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/700Wide.png)]
(milestone 3)
### [optional change]
- you can center the whole calculator
```
margin: auto
```